src/apiCalls.js

```diff
@@ -0,0 +1,26 @@
+const fetchUsersData = () => {
```

Comment:
We may be able to refactor this to the interpolation when we create instances of our Classes